src/routes/examples/react/resource-helper.mdx

```diff
@@ -0,0 +1,215 @@
+# The Resource Helper
+
+This example builds on the previous one concerning fixtures but also
+includes:
+
+* Configuration change to expose the `Country` model as a resource
+* The data class is enhanced to implement all CRUD
+* Jest tests assert all resource requests
+
+## Mirage Configuration
+
+```js
+// src/mirage.js
+
+import { Model, Server } from '@miragejs/server';
+
+let startMirage = function({ environment = 'test' } = {}) {
+  return new Server({
+    environment,
+
+    // fixture data will be loaded when the
+    // Mirage server starts up
+    fixtures: {
+      countries: [
+        { id: 1, name: 'China' },
+        { id: 2, name: 'India' },
+        { id: 3, name: 'United States' }
+      ],
+    },
+
+    // declare the models that Mirage needs
+    // to know how to serve
+    models: {
+      country: Model
+    },
+
+    baseConfig() {
+      // routes below `this.urlPrefix` assignment are
+      // expected to resolve to `https://some.example.com`
+      this.urlPrefix = 'https://some.example.com';
+
+      // ------------------------------ 🎉🎉🎉 New Section 🎉🎉🎉
+      // EXAMPLE: the resource shorthand will expose the
+      // GET/POST/DELETE/PATCH endpoints;
+      // e.g. GET/POST of https://some.example.com/countries
+      this.resource('countries');
+      // ---------------------------------- 🎉🎉🎉 End New 🎉🎉🎉
+
+      // all other API requests are passed
+      // through to their destinations
+      this.passthrough();
+    }
+  });
+};
+
+export { startMirage };
+```
+
+## CRUD Class
```

Comment:
I don't love this _CRUD_ class.  I am trying to demonstrate to readers that their API calls will be in some sort of class/function similar to this.  Don't want the reader to be mired in the implementation, except to understand that this is the default MirageJs serializer behaviour.